src/components/BookCards.js

```diff
@@ -22,36 +23,39 @@ const BookCard = () => {
         <div key={book.item_id} className="book-card">
           <div className="book-info">
             <span className="book-category">{book.category}</span>
-            <h2>{book.title}</h2>
-            <span>{book.author}</span>
+            <h2 className="title">{book.title}</h2>
+            <span className="author">DANIEL YERIMAH</span>
             <div className="container">
-              <ul>
+              <ul className="utils">
                 <li>
-                  <button id={book.item_id} type="button">
+                  <button id={book.item_id} className="btn border-right" type="button">
                     Comments
                   </button>
                 </li>
                 <li>
-                  <button id={book.item_id} type="button">
-                    Edit
+                  <button id={book.item_id} className="btn border-right" type="button" onClick={removeHandler}>
+                    Remove
                   </button>
                 </li>
                 <li>
-                  <button id={book.item_id} type="button" onClick={removeHandler}>
-                    Remove
+                  <button id={book.item_id} className="btn" type="button">
+                    Edit
                   </button>
                 </li>
               </ul>
             </div>
           </div>
           <div className="progress">
-            <span>64%</span>
-            <span>completed</span>
+            <img className="progress-icon" src={StatusIcon} alt="circular status icon" />
+            <div className="metrics">
+              <span className="percent">64%</span>
+              <span className="state">completed</span>
+            </div>
           </div>
           <div className="view-progress">
-            <span>Current chapter</span>
-            <h3>Chapter 5</h3>
-            <button id={book.item_id} type="button">Update progress</button>
+            <span className="cur-chapter">CURRENT CHAPTER</span>
+            <h3 className="bookchapter">Chapter 5</h3>
+            <button className="update-btn" id={book.item_id} type="button">UPDATE PROGRESS</button>
```

Comment:
- [ ] Add this missing horizontal line :arrow_down: 

![Screenshot](https://user-images.githubusercontent.com/57562869/156856786-4fa21119-3b5f-44b3-9bde-49dcf0413877.png)